src/components/PastTripForm.spec.js

```diff
@@ -0,0 +1,41 @@
+import { render, screen } from '@testing-library/react';
+import PastTripNotes from './PastTripNotes';
+import userEvent from '@testing-library/user-event';
+
+describe('PastTripNotes', () => {
+  it('renders an input with a placeholder', () => {
+    render(
+      <PastTripNotes
+        labelText="Things to remember"
+        placeholder="Enter your notes..."
+        name="notes"
+      />
+    );
+
+    const input = screen.getByLabelText(/Things to remember/i);
+    expect(input).toHaveAttribute('placeholder', 'Enter your notes...');
+
+    const button = screen.getByRole('button');
+    expect(button).toBeInTheDocument();
+  });
+
+  it('renders a form with the name "Add notes"', () => {
+    render(<PastTripNotes />);
+
+    const form = screen.getByRole('form', { name: 'Add notes' });
+    expect(form).toBeInTheDocument();
+  });
+
+  it('submits form data when field is filled out', () => {
+    const handleNewNote = jest.fn();
+    render(<PastTripNotes onHandleNewNote={handleNewNote} />);
+
+    const input = screen.getByLabelText(/Things to remember/i);
+    const button = screen.getByRole('button');
+
+    userEvent.type(input, '{enter}');
+    userEvent.click(button);
+
+    expect(handleNewNote).not.toHaveBeenCalledWith();
+  });
```

Comment:
Since you expect that `handleNewNote` has not been called, your a not testing whether the form is submitted, but whether it is not submitted.
Do line 36 actually type something into the input element?